### PR TITLE
Add v2.0.0 upgrade instructions

### PR DESCRIPTION
**Pull request by JakeHartnell**

### CODE REVIEW

File: stargaze-1/v2_0_0_UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# v2.0.0 Upgrade
+
+The upgrade is scheduled for block `1317621`, which should be about _15:30PM UTC on 25th January 2022_.
```
Comment (jhernandezb) [inline stargaze-1/v2_0_0_UPGRADE.md:3]:
```suggestion
The upgrade is scheduled for block `1317621`, which should be about _05:30PM UTC on 25th January 2022_.
```
or maybe just 17:30

File: stargaze-1/v2_0_0_UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+These instructions assume you are running Cosmovisor. Most mainnet validators are running Cosmovisor, and [a setup guide can be found here](https://docs.junonetwork.io/validators/setting-up-cosmovisor).
+
```
Comment (jhernandezb) [inline stargaze-1/v2_0_0_UPGRADE.md:6]: Can we add a warning saying that cosmovisor is configured to take a full backup unless `UNSAFE_SKIP_BACKUP=true`

Comment (JakeHartnell) [reply stargaze-1/v2_0_0_UPGRADE.md:6]: Yes.

Comment (JakeHartnell) [reply stargaze-1/v2_0_0_UPGRADE.md:6]: Added.

**Review by jhernandezb — APPROVED** [review]

**Pull request merged**